custom/enikshay/user_setup.py

```diff
@@ -155,7 +155,7 @@ def save_user_callback(sender, couch_user, **kwargs):
         changed = set_issuer_id(commcare_user.domain, commcare_user) or changed
         changed = add_drtb_hiv_to_dto(commcare_user.domain, commcare_user) or changed
         if changed:
-            commcare_user.save()
+            commcare_user.save(fire_signals=False)
```

Comment:
Dont know much about how this would effect but do we need any signals that should be triggered?

Reply:
I think this will stop it recursively calling this method again, as this is the one called by the signal. 
👍 I think this could have been a better fix to the "device id keeps updating and getting sent to BETS" issue that I fixed rather convolutedly before.